0-SCore/Harmony/NPCFeatures/EntityAlivePatcher.cs

```diff
@@ -48,13 +48,13 @@ public class EntityAlivePatcher
         [HarmonyPatch("DamageEntity")]
         public class EntityAliveDamageEntity
         {
-            private static bool Prefix(global::EntityAlive __instance, DamageSource _damageSource)
+            private static bool Prefix(global::EntityAlive __instance, ref int __result, DamageSource _damageSource)
```

Comment:
```suggestion
            private static bool Prefix(global::EntityAlive __instance, DamageSource _damageSource)
```

Reply:
Sorry, I thought I needed to also set the result when I bypassed the original method. It turns out I don't, and I forgot to remove that parameter.